File: lab 3/pop.cpp
#include <iostream>
using namespace std;

const int SIZE = 5;
int stack[SIZE] = {10, 20, 30};
int top = 2;

void pop()
{
    if (top == -1)
    {
        cout << "Stack Underflow" << endl;
        return;
    }

    cout << stack[top] << " popped from stack" << endl;
    top--;
}

int main()
{
    pop();
    pop();

    return 0;
}

File: lab 3/pop.java
public class pop {
    static int[] stack = { 10, 20, 30 };
    static int top = 2;

    static void popStack() {
        if (top == -1) {
            System.out.println("Stack Underflow");
            return;
        }

        System.out.println(stack[top] + " popped from stack");
        top--;
    }

    public static void main(String[] args) {
        popStack();
        popStack();
    }

}
